apps/dashboard/src/components/primitives/tag-input.tsx

```diff
@@ -52,21 +79,37 @@ const TagInput = forwardRef<HTMLInputElement, TagInputProps>((props, ref) => {
   };
 
   return (
-    <div className="flex flex-col space-y-2">
-      <InputField>
-        <Input
-          ref={ref}
-          type="text"
-          value={inputValue}
-          onKeyDown={handleKeyDown}
-          className={cn('flex-grow', className)}
-          placeholder="Type a tag and press Enter"
-          onChange={(e) => setInputValue(e.target.value)}
-          {...rest}
-        />
-      </InputField>
+    <div className="flex flex-col space-y-2" ref={divRef}>
+      {showAddButton && isFocussed && (
+        <InputField>
+          <Input
+            ref={ref}
+            type="text"
+            value={inputValue}
+            onKeyDown={handleKeyDown}
+            className={cn('flex-grow', className)}
+            placeholder="Type a tag and press Enter"
+            onChange={(e) => setInputValue(e.target.value)}
```

Comment:
I noticed that somehow the font-size and placeholder font-size are not aligned with the rest of inputs
![Screenshot 2024-10-22 at 00 38 17](https://github.com/user-attachments/assets/e567a78d-83d0-44b6-bff2-a7cb90f79af5)